serialization/astronomy.proto

```diff
@@ -17,7 +17,7 @@ message GravityModel {
     optional string reference_angle = 8;
     optional string angular_frequency = 9;
     // Oblate body.
-    optional string j2 = 5;
+    optional string j2 = 5;  // TODO(phl): Change to a double.
```

Comment:
I won't change the KSP configs, it's just internal to the C++.  And I trust protobuf to know how to convert back and forth.  The PR is mostly ready anyway.